python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -769,7 +707,9 @@ def scheduler_class(self) -> Optional[str]:
     # run coordinator
 
     @property
-    def run_coordinator(self) -> "RunCoordinator":
+    def run_coordinator(
+        self,
+    ):  # FIXME(deepyaman): -> "dagster._core.run_coordinator.RunCoordinator":
```

Comment:
I need to debug why a small handful of these aren't working; will look into it further if get alignment on using beartype.